tests/helpers.py

```diff
@@ -2233,3 +2234,58 @@ def validate_pods_are_running_and_not_restarted(
     logger.error(f"Pod is in {pod_state} state and restart count of pod {restart_count}")
     logger.info(f"{pod_obj}")
     return False
+
+
+def delete_objs(pvc_objs):
+    """
+    Delete pvc objects
+
+    Args:
+        pvc_objs (list): pvc objects to delete
+    """
+    for obj in pvc_objs:
+        obj.delete()
+        obj.ocp.wait_for_delete(obj.name)
```

Comment:
You should not wait for deletion to complete if you are running at scale. At the end, you should ensure all objects were deleted.

Reply:
Thanks.  I will do this in PR 2236.